src/include/statistics/abstract_metric_new.h

```diff
@@ -0,0 +1,58 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// abstract_metric.h
+//
+// Identification: src/statistics/abstract_metric.h
```

Comment:
You're calling this file abstract_metric_new.h...

Actually, I'm not sure that's a good name. Do we still need the old abstract_metric.h?